build/config.cmake

```diff
@@ -159,6 +123,46 @@ IF (${ESCARGOT_OUTPUT} STREQUAL "shared_lib")
     SET (ESCARGOT_LDFLAGS_ANDROID ${ESCARGOT_LDFLAGS_ANDROID} -shared)
 ENDIF()
 
+IF (NOT DEFINED ESCARGOT_LIBICU_SUPPORT)
+  SET (ESCARGOT_LIBICU_SUPPORT ON)
+ENDIF()
+
+# LIBICU SUPPORT
+IF (${ESCARGOT_LIBICU_SUPPORT} STREQUAL "ON")
+    SET (ESCARGOT_DEFINITIONS_LINUX ${ESCARGOT_DEFINITIONS_LINUX} -DENABLE_ICU)
+    SET (ESCARGOT_DEFINITIONS_DARWIN ${ESCARGOT_DEFINITIONS_DARWIN} -DENABLE_ICU)
+    SET (ESCARGOT_DEFINITIONS_TIZEN ${ESCARGOT_DEFINITIONS_TIZEN} -DENABLE_ICU)
+
+    # LINUX LIBRARIES
+    IF (${ESCARGOT_ARCH} STREQUAL "x64" OR ${ESCARGOT_ARCH} STREQUAL "x86" OR ${ESCARGOT_ARCH} STREQUAL "arm")
+        PKG_CHECK_MODULES (ICUI18N REQUIRED icu-i18n)
```

Comment:
Checking for each target is omitted. It would be clear if we check each target (`ESCARGOT_HOST`) before executing `PKG_CHECK_MODULES`.
For example, this `IF` statement needs to be wrapped with another `IF` statement, `IF(${ESCARGOT_HOST} STREQUAL "linux")`.

Reply:
Others seem good to me.